content/en/docs/tasks/tools/install-kubeadm.md

```diff
@@ -52,22 +52,20 @@ route, we recommend you add IP route(s) so Kubernetes cluster addresses go via t
 
 ### Master node(s)
 
-| Protocol | Direction | Port Range | Purpose                 |
-|----------|-----------|------------|-------------------------|
-| TCP      | Inbound   | 6443*      | Kubernetes API server   |
-| TCP      | Inbound   | 2379-2380  | etcd server client API  |
-| TCP      | Inbound   | 10250      | Kubelet API             |
-| TCP      | Inbound   | 10251      | kube-scheduler          |
-| TCP      | Inbound   | 10252      | kube-controller-manager |
-| TCP      | Inbound   | 10255      | Read-only Kubelet API   |
+| Protocol | Direction | Port Range | Purpose                 | Source Components         |
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Inbound   | 6443*      | Kubernetes API server   | All                       |
+| TCP      | Inbound   | 2379-2380  | etcd server client API  | kube-apisever, etcd peers |
```

Comment:
etcd server and peer ports should probably be broken out. While it doesn't really matter when etcd is co-located with the control plane components, if etcd is broken out the firewall requirements are different for each port.

Reply:
`s/kube-apisever/kube-apiserver`

Reply:
> While it doesn't really matter when etcd is co-located with the control plane components

Mostly it is true. But kubeadm does provide a option to install etcd together with control plane by etcd manifest. So I prefer to keep it. @luxas WDYT?